config/i18n/locales/german/translations.json

```diff
@@ -0,0 +1,97 @@
+{
+  "buttons": {
+    "forum": "Forum",
+    "donate": "Spende",
+    "load-more-articles": "Weitere Artikel laden"
+  },
+  "search": {
+    "label": "Suche",
+    "accessible-name": "Sende deine Suchanfrage ab",
+    "placeholder": "Mehr als 8.000 Tutorials durchsuchen",
```

Comment:
This is one string that will be different across all the publications. The English string refers to the number of published articles.

Could we change this placeholder to be something more generic like "Search our tutorials" without a number of published articles?

Feel free to use whatever wording you think is best.

Reply:
Of course.

"placeholder": "Suche in unseren Tutorials",